src/aiodynamo/client.py

```diff
@@ -930,6 +943,27 @@ async def batch_write(
             result[table] = BatchWriteResult(undeleted_keys, unput_items)
         return result
 
+    async def transact_write_items(
+        self,
+        items: Sequence[Union[Put, Update, Delete, ConditionCheck]],
+        *,
+        request_token: Optional[str] = None,
+    ) -> None:
+        if len(items) == 0:
+            raise TransactionCanceled(
+                "TransactWriteItems must have at least 1 operation"
+            )
+        if len(items) > 25:
+            raise TransactionCanceled(
+                "TransactWriteItems must have a maximum of 25 operations"
+            )
```

Comment:
I would prefer two separate exceptions for these cases which are distinct so you don't need to check the string to figure out what went wrong. Feel free to come up with better names but maybe something like `TransactionEmpty` and `TransactionTooLarge`